JavaScript/CodingConventions.md

```diff
@@ -477,11 +499,37 @@ function makeCookies(flavor) {
 
 We use JSDoc style comments above function declarations and at the top of files because they make code clear and easy to read, and we'd like to be able to generate JavaScript documentation at some point.
 
+The description for parameters is optional; description for `@returns` statement is not optional. Also it's encouraged to have description and params separated by newline.
```

Comment:
Ok, let's talk about that more. Do you think it shouldn't be mandatory? IMHO sometimes it's easier to write what function returns than writing function's description.